content/en/apps/guides/integrations/oppia.md

```diff
@@ -0,0 +1,132 @@
+
+```
+Medic-conf --instance=<instance> convert-app-forms upload-app-forms -- <form1> <form2>
+```
+
+or
+
+```
+Medic-conf --instance=<instance> convert-app-forms upload-app-forms
+```
+
+The image below shows an example of an XLSForm configured for the Educational Modules:
+
+{{< figure src="xls-modules.png" link="xls-modules.png" class="left col-12 col-lg-9" >}}
+
+### Integration with OppiaMobile using weblinks
+
+The CHT application makes use of weblinks to direct the user to the OppiaMobile application. Each task in the CHT has a weblink configured to point to a specific course in OppiaMobile. The weblinks are configured in the forms as a button, which, when clicked or tapped, redirects the user accordingly, depending on the installation status of the OppiaMobile application and the respective course. The weblinks are configured in each of the XLSForms that are triggered by a selected task as illustrated in the image below: 
+
+{{< figure src="weblink.png" link="weblink.png" class="left col-12 col-lg-9" >}}
+
+<br clear="all">
+
+```[<span style='background-color: #648af7; color:white; padding: 1em; text-decoration: none;border-radius: 8px; '>Open Oppia Mobile</span>]``` represents the button styling and label.
+
+`https://staging.cha.oppia-mobile.org/view?course=introduction-to-covid-19` represents the weblink, where `course=introduction-to-covid-19` specifies the name of the course to be launched in OppiaMobile as configured on Moodle.
+
+This image shows the outcome of the button configuration:
+
+{{< figure src="weblink-config-outcome.png" link="weblink-config-outcome.png" class="left col-7 col-lg-4" >}}
+
+
+### Tasks
+
+[Tasks]({{< ref "apps/reference/tasks" >}}) are a set of actions available to users from the task tab. Selecting a task opens up a specific form that completes a workflow. Tasks are available within a given timeframe, after which they expire and the user is unable to view or do them. Tasks are defined as an array of objects in a tasks.js file under the project folder, with each task following the task schema. The required properties of a task include:
+
+- Name - unique identifier of the task;
+- Title - displays the workflow to be completed for a contact;
+- AppliesTo - determines if the task is emitted per contact or report;
+- ResolvedIf - conditions to mark a task as resolved/completed;
+- Events - specifies the timeframe of a task;
+- Actions - specifies the forms accessed by the user and allows injecting content from previous submissions.
+
+On completing the task configuration, the following command is used to compile and upload  the applications settings (tasks, targets, contact-summary, form properties):
+
+```Medic-conf --instance=<instance> compile-app-settings upload-app-settings```
+
+The image below illustrates an example of a task configured for the educational modules:
+
+{{< figure src="code-tasks.png" link="code-tasks.png" class="left col-12 col-lg-9" >}}
```

Comment:
If it's possible and wouldn't take too much work, I think it would be much better to have code snippets here instead of images, but I wouldn't let it hold us back.  We should be able to just copy/paste from those files. 

With snippets, it's much easier to make changes/updates if necessary, and it'll be easier to make sure they are consistent.

Reply:
@IreneAchola, are you able to support here? As Michael said, I don't think we should keep this from pushing these pages but it would be a nice update

Reply:
Yes, I am able to.